WeakAuras/Transmission.lua

```diff
@@ -726,16 +729,20 @@ local function recurseStringify(data, level, lines)
   end
 end
 
-function WeakAuras.DisplayToTableString(id)
+function WeakAuras.DisplayAuraToString(id)
   local data = WeakAuras.GetData(id)
-  if(data) then
-    local lines = {"{"};
-    recurseStringify(data, 1, lines);
-    tinsert(lines, "}")
-    return table.concat(lines, "\n"):gsub("|", "||");
+  if data then
+    return WeakAuras.SerializeTable(data)
   end
 end
 
+function WeakAuras.SerializeTable(data)
+  local lines = {"{"}
+  recurseStringify(data, 1, lines)
+  tinsert(lines, "}")
+  return table.concat(lines, "\n"):gsub("|", "||")
```

Comment:
the gsub here is intended specifically so that pipes don't get treated as magic characters in a blizzard fontstring. It's probably best to move this to DisplayAuraToString.